pennylane/ops/op_math/exp.py

```diff
@@ -210,24 +230,30 @@ def _queue_category(self):
             return "_ops"
         return None
 
-    # pylint: disable=arguments-renamed, invalid-overridden-method
+    # pylint: disable=invalid-overridden-method, arguments-renamed
     @property
     def has_decomposition(self):
-        if isinstance(self.base, Tensor):
-            all_wires = [obs.wires for obs in self.base.obs]
-            if len(all_wires) != len(set(all_wires)):
-                raise NotImplementedError(
-                    "Unable to determine if the exponential has a decomposition "
-                    "when the base operator is a Tensor object with overlapping wires. "
-                    f"Received base {self.base}."
-                )
-        return math.real(self.coeff) == 0 and qml.pauli.is_pauli_word(self.base)
+        # TODO: Support nested sums in method
```

Comment:
Maybe a good idea to add a shortcut story for this TODO

Reply:
+1.  TODO's rarely ever get done when placed in code.

Reply:
Where should I add the ticket (in which epic)? @albi3ro

Reply:
"Evolution operator has smart decompositions"?

Reply:
Ok! It would be nice to have an epic for these TODOs though

Reply:
Added `TODOs` epic in Shortcut and created a story with this.

I don't want to add a story in `Evolution operator has smart decompositions` because this epic is linked to Productboard, and I want to mark it as done as soon as this PR is merged.